v2/go.mod

```diff
@@ -1,5 +1,3 @@
 module github.com/googleapis/gax-go/v2
 
-go 1.12
```

Comment:
I thought this was auto-added?

Reply:
Only when you `go mod init` or `go mod tidy` on an empty `go.mod`.